frameworks/Rust/actix/src/main.rs

```diff
@@ -20,7 +20,7 @@ async fn json() -> HttpResponse {
         message: "Hello, World!",
     };
     let mut body = BytesMut::with_capacity(SIZE);
-    serde_json::to_writer(Writer(&mut body), &message).unwrap();
+    simd_json::to_writer(Writer(&mut body), &message).unwrap();
```

Comment:
it would be better to use `simd-json-derive::Serialize` and `message.json_write` it is 2-3x faster then serde::Serialize.

`simd_json::to_writer` is a compatibility method that implements the slower serde-trait for drop-in replacement w/o code changes. Given it's a benchmark code changes are OK I think especially if it's about 2 lines :D.

That said the `simd-json-derive::Serialize` is still young and only exists as a git dependency, not as a crate yet.

Reply:
@Licenser thanks for the tip!